wal/wal.go

```diff
@@ -273,6 +273,11 @@ func (w *FileWAL) Replay(handler func(tx uint64, record *walpb.Record) error) er
 		return fmt.Errorf("read last index: %w", err)
 	}
 
+	// FirstIndex and LastIndex returns zero when there is no WAL files.
+	if firstIndex == 0 || lastIndex == 0 {
```

Comment:
There's already a directory check in `db.go:166` I think we should just need to update that instead no?

Reply:
db.go:166 actually checks for the existence of database directory not the wal. Also we may not able to check the existence of wal dir because creating a WAL instance[1] would always create a `wal` directory[2].

[1] https://github.com/polarsignals/frostdb/blob/363a6ad4c48da19af87f94ca0a6dcb1283e0e01b/db.go#L303
[1]https://github.com/tidwall/wal/blob/0fa32977a196086591e6eace7bdf13f67bc305aa/wal.go#L148

Reply:
Great catch! Okay then this seems like a reasonable approach to me!